packages/rpc-core/src/rpc-methods/getTokenLargestAccounts.ts

```diff
@@ -0,0 +1,31 @@
+import { Base58EncodedAddress } from '@solana/keys';
+
+import { Commitment, RpcResponse } from './common';
+
+type GetTokenLargestAccountsApiResponse = RpcResponse<{
+    /** the address of the token account */
+    address: Base58EncodedAddress;
+    /** the raw token account balance without decimals, a string representation of u64 */
+    amount: string;
```

Comment:
Sigh. I wonder if it's worth creating an opaque type for this now. Basically, this is ‘bigint as a string.’ It would be worth knowing when you have one of these types so that you know for sure it's parseable by `BigInt(...)`.